modules/tinymce/src/themes/silver/main/ts/ui/sidebar/Sidebar.ts

```diff
@@ -94,9 +94,20 @@ const makeSidebar = (panelConfigs: SidebarConfig) => SlotContainer.sketch((parts
   ])
 }));
 
-const setSidebar = (sidebar: AlloyComponent, panelConfigs: SidebarConfig) => {
+const setSidebar = (sidebar: AlloyComponent, panelConfigs: SidebarConfig, showSidebar?: string) => {
   const optSlider = Composing.getCurrent(sidebar);
-  optSlider.each((slider) => Replacing.set(slider, [ makeSidebar(panelConfigs) ]));
+
+  optSlider.each((slider) => {
+    Replacing.set(slider, [ makeSidebar(panelConfigs) ]);
+
+    // Show the default sidebar
+    if (Type.isString(showSidebar)) {
+      Composing.getCurrent(slider).each((slotContainer) => {
+        SlotContainer.showSlot(slotContainer, showSidebar);
+      });
+      Sliding.grow(slider);
```

Comment:
This will still cause the sidebar to animate which we don't want per the Jira requirements. We probably need to implement a `growImmediately` API in the alloy sliding behaviour to solve this.

The other issue is that we probably don't want to grow if there is no slot container found. So we should put this beside the `SlotContainer.showSlot` call instead.